test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fixtures.messages.sections.transportArranger
+
+import fixtures.messages.{BaseEnglish, BaseMessages, BaseWelsh, i18n}
+import models.sections.transportArranger.TransportArranger
+import models.sections.transportArranger.TransportArranger.{GoodsOwner, Other}
+
+object TransportArrangerNameMessages {
+
+  sealed trait ViewMessages extends BaseMessages { _: i18n =>
+    def title()(implicit transportArranger: TransportArranger) = transportArranger match {
+      case GoodsOwner => titleHelper("What is the goods owner's business name?")
+      case Other => titleHelper("What is the consignee's business name?")
+    }
+
+    def heading()(implicit transportArranger: TransportArranger) = transportArranger match {
+      case GoodsOwner => "What is the goods owner's business name?"
+      case Other => "What is the consignee's business name?"
```

Comment:
This is wrong. It should be What is the transport arranger's business name?

Reply:
Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71